nvtabular/dask/io.py

```diff
@@ -239,27 +262,48 @@ class ParquetDatasetEngine(DatasetEngine):
         Dask-based version of cudf.read_parquet.
     """
 
-    def __init__(self, *args, row_groups_per_part=None):
+    def __init__(self, *args, row_groups_per_part=None, legacy=False):
```

Comment:
Whats the use case for this legacy parameter?

Reply:
This is just a switch that someone running with 0.17.1 can use to easily compare legacy pyarrow vs their new datasets API.  The argument is certainly not necessary, but I just left it in for now.